Makefile

```diff
@@ -64,8 +65,29 @@ proxy-h:
 proxy-m:
 	consul connect proxy -service minio-local -upstream minio:9000 -log-level debug
 # to-postgres
-proxy-p:
+proxy-pg:
 	consul connect proxy -service postgres-local -upstream postgres:5432 -log-level debug
+
 # to-presto
-proxy-p:
-	consul connect proxy -service postgres-local -upstream presto:8080 -log-level debug
+proxy-presto:
+	consul connect proxy -service presto-local -upstream presto:8080 -log-level debug
+
+download-presto-cli:
+	curl https://repo1.maven.org/maven2/io/prestosql/presto-cli/${PRESTO_VERSION}/presto-cli-${PRESTO_VERSION}-executable.jar -o ./${presto}
+	chmod +x ./${presto}
+
+presto-cli:
```

Comment:
```suggestion
presto-cli: proxy-presto
```

Reply:
Proxy-presto stdout is attached to the terminal. I would suggest running these two commands `presto-cli` and `proxy-presto` in separate terminals.